src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/hiddenfield/Example1.java

```diff
@@ -0,0 +1,29 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HiddenField"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.hiddenfield;
+
+// xdoc section -- start
+public class Example1 {
+
+  private String field;
+  private String testField;
+
+  public Example1(String testField) { // violation
+  }
+  public void method(String param) { // OK
```

Comment:
We should not have ok comments without extra explanations.
Please remove in all examples.

Reply:
Done